app/models/statement_item.rb

```diff
@@ -0,0 +1,52 @@
+class StatementItem < ApplicationRecord
```

Comment:
curious around the naming decision (I guess that's what we landed on with the analysis) vs line item?

Reply:
I'm not massively invested in either line or item but they both mean the same thing here so using both doesn't feel like it makes sense.

Reply:
In the analysis, we talked about this just being a shorter maybe name.

But I wonder, maybe we should instead call it `StatementDeclaration`, as it closely describes the many-to-many relationship.

Reply:
I think that sounds reasonable, the only thing that makes me hesitant is that there is a `state` on the record, so happy to stick to items for now.